### PR TITLE
Fix/154/sponsorship-section-style-improvement

### PR DESCRIPTION
**Pull request by AmirAgassi**
🛠️[[Fix] sponsorship section style improvement #154](https://github.com/LaurierHawkHacks/Landing/issues/154)

🔍 **What's Included**
- Added horizontal sliders for a cleaner display of sponsors and partners in `Sponsor.section.tsx`.
- Updated the Taskade logo to an uncropped version.

📁 Files Affected:
- `src/assets/logos/taskade.png`
- `src/components/sections/Sponsor.section.tsx`

### CODE REVIEW

**Comment by aidantrabs** [comment]
womp womp

image

**Review by GabeDiniz — COMMENTED** [review]
The bar scroll bar is a little wonky looking..
May I suggest, we could either change the styling of the scroll bar (https://www.geeksforgeeks.org/how-to-change-style-of-scrollbar-using-tailwind-css/) 
OR make it so when there's overflow, its an animated carousel that infinitely goes through the sponsors? <-- this one might be a pain in the ass to do though
image

File: src/components/sections/Sponsor.section.tsx [hunk 1]
```diff
@@ -1,199 +1,278 @@
...
+              (totalAnimationTime / 2);
+            currentPosition =
+              scrollDistance - easeInOutCubic(progress) * scrollDistance;
+            lastAnimationPhase = "backward";
+          }
+
+          carousel.style.transform = `translateX(-${currentPosition}px)`;
+          animationFrameIds.set(
+            carousel,
+            requestAnimationFrame(updatePosition)
```
Comment (juancwu) [inline src/components/sections/Sponsor.section.tsx:84]: you know its real when you call `requestAnimationFrame`

**Review by juancwu — APPROVED** [review]
lgtm

File: src/components/sections/Sponsor.section.tsx (hunk 1 above)
Comment (aidantrabs) [reply src/components/sections/Sponsor.section.tsx:84]: > you know its real when you call `requestAnimationFrame`

ong bro, 10x react dev right here

**Review by GabeDiniz — APPROVED** [review]
This is crazy bro.. good shit

**Review by teoh4770 — APPROVED** [review]
W

**Review by juancwu — APPROVED** [review]

**Pull request merged**